src/components/ConnectButton/index.tsx

```diff
@@ -18,7 +17,6 @@ let providerName
 const wallets = getSupportedWallets()
 
 export const onboard = Onboard({
-  dappId: BLOCKNATIVE_KEY,
```

Comment:
is it possible to make it conditional? 
at this moment, we know which network is the user using. So we can do something like this:
`[EWC,Volta].includes(network) ? null : BLOCKNATIVE_KEY.`

Reply:
In general BLOCKNATIVE_KEY is adding some console errors and we are not really using it.
They usually print a rate limit error and we are not using any functionality that needs a KEY.

I think we should remove it for all networks